rmgpy/cantherm/molpro.py

```diff
@@ -260,58 +260,89 @@ def loadConformer(self, symmetry=None, spinMultiplicity=0, opticalIsomers=1, sym
         f.close()
         return Conformer(E0=(E0*0.001,"kJ/mol"), modes=modes, spinMultiplicity=spinMultiplicity, opticalIsomers=opticalIsomers)
 
-    def loadEnergy(self,frequencyScaleFactor=1.):
+    def loadEnergy(self, frequencyScaleFactor=1., model_chemistry=''):
```

Comment:
Why is `model_chemistry` needed here? Shouldn't we be able to read this information from the log file?

Reply:
We sometimes use a similar level of theory for model_chemistry, so to avoid errors I think it's best to pass it